week4:JSweek2/5.js

```diff
@@ -0,0 +1,5 @@
+var z = '7.25';
+console.log(z);
+var a = Math.ceil(7.25);
```

Comment:
You could use the variable you defined above, eg `Math.ceil(z)`.
Also, did you want to use `ceil` or `round` here?

Reply:
true, it does not necessarily ask me to round up. just round. need to simplify my thinking process :)